testsuite/features/support/commonlib.rb

```diff
@@ -70,7 +70,7 @@ def product_version
 
 def use_salt_bundle
   # Use venv-salt-minion in Uyuni, or SUMA Head and 4.3
```

Comment:
I'd update this comment to reflect that we now use venv-salt-minion across all our product versions